packages/node/test/setup/e2e/testing.ts

```diff
@@ -0,0 +1,25 @@
+import fs from 'fs';
+import { Request } from '@api3/operation';
+import { buildChainConfig, buildProvider, getDeployerIndex } from './utils';
+import { deployAirnodeRrp, makeRequests } from './deployment';
+import { buildConfig, operation } from '../../fixtures';
+
+export const increaseTestTimeout = (timeoutMs = 45_000) => jest.setTimeout(timeoutMs);
+
+export const deployAirnodeAndMakeRequests = async (filename: string, requests?: Request[]) => {
+  const deployerIndex = getDeployerIndex(filename);
+  const deployConfig = operation.buildDeployConfig(requests ? { deployerIndex, requests } : { deployerIndex });
```

Comment:
This may need to be changed at some point to be a bit more flexible with configuring the fixtures, but for now it's fine